spec/openapi.yaml

```diff
@@ -262,30 +153,17 @@ paths:
                 type: object
                 properties:
                   job:
-                    $ref: "#/components/schemas/OctoJob"
+                    $ref: "#/components/schemas/Job"
                   progress:
-                    $ref: "#/components/schemas/OctoProgress"
+                    $ref: "#/components/schemas/Progress"
                   state:
-                    enum:
-                      [
-                        "Operational",
-                        "Printing",
-                        "Pausing",
-                        "Paused",
-                        "Cancelling",
-                        "Error",
-                        "Offline",
-                      ]
+                    $ref: "#/components/schemas/State"
         403:
           $ref: "#/components/responses/Unauthorized"
         409:
           description: File not found or wrong format.
```

Comment:
How about the error format?
It would be in header or query 
how is the format of the answer

in:
- header
- query

    get:
      parameters:
        - in: header
          name: error-format
          schema:
            type: string
            default: string
            enum:
              - json
              - html
              - string
              
        - in: query
          name: error-format
          schema:
            schema:
            type: string
            default: string
            enum:
              - json
              - html
              - string

Reply:
I'm waiting for the answer from MINI team. In general I think it would be good to pass additional header variables in HTTP error responses 30x, 40x, 50x.